src/workerd/io/io-context.c++

```diff
@@ -824,6 +824,15 @@ kj::Date IoContext::now() {
   return now(getCurrentIncomingRequest());
 }
 
+double IoContext::performanceNow() {
+  // If there are no incoming requests, this should always return 0.0.
+  // Otherwise, the time origin for an incoming request should always be set.
+  if (!incomingRequests.empty()) {
+    return (now() - kj::UNIX_EPOCH) / kj::NANOSECONDS * 1e-6;
```

Comment:
This is returning milisecs, right? Would this be equivalent (and more readable)?

```suggestion
    return (now() - kj::UNIX_EPOCH) / kj::MILLISECONDS;
```

Reply:
I used this pattern specifically because it is used by the internal `UnsafePerformanceApi` binding, basically following @kentonv's lead on it :-)

Reply:
Please apply @dom96 change. Otherwise this is revealing information that Date.now() did not previously reveal.

Reply:
Actually can we just not add a new method here at all? Use the existing one for Date.now().

Reply:
https://github.com/cloudflare/workerd/pull/617